js/cards.js

```diff
@@ -116,7 +116,7 @@ $(document).ready(function(){
     let newUrl= url.split("/")
     let key=newUrl.pop();
     let num=key.split("=")[1]||1;
-    if (num===1) {
+    if (num==1) {
```

Comment:
should be `===`. If the types are wrong, you need to be explicit, you can change `1` to `"1"` or use parseInt